inspector/src/components/actionTabs/tabs/propertyGrids/animations/animationCurveEditorComponent.tsx

```diff
@@ -197,7 +200,7 @@ export class AnimationCurveEditorComponent extends React.Component<
 
   setAxesLength() {
     let length = 20;
-    let newlength = Math.round(this._canvasLength * this.state.scale); // Check Undefined, or NaN

```

Comment:
Was this comment addressed in any way?

Reply:
ah, yes. I need to remove it. The problem was actually happening because I wasn't rounding the value to int. So the array would break. The Math.rounds resolved the issue. Thanks for noticing this :)